### DISCUSSION TITLE
Utility focus ring not working

### DISCUSSION THREAD

**Discussion by FilipePfluck** [chakra-ui/panda#873] [Q&A]
### Description

I tried to create a focusRing utility like this: 

focusRing: {
      values: ['1px', '2px', '4px'],
      shorthand: 'ring',
      transform(value: string) {
        return {
          _focusVisible: {
            outlineWidth: value,
            outlineStyle: 'solid',
            outlineColor: 'blue.500',
            outlineOffset: value,
          },
        }
      },
    },
    
  when I  try to use it anywhere, Vscode does suggest this property. But it does not work, instead, it shows the default browser focus indicator. If I use just 
  
  _focusVisible: {
            outlineWidth: '2px',
            outlineStyle: 'solid',
            outlineColor: 'blue.500',
          },
          
   then it works

### Link to Reproduction

https://github.com/FilipePfluck/my-components

### Steps to reproduce

go to globalCss.ts and try substituting the focus indicator css with the custom property `ring`
run storybook
see it doesn't work

### JS Framework

React (TS)

### Panda CSS Version

0.4.0

### Operating System

- [ ] macOS
- [ ] Windows
- [X] Linux

**Comment by FilipePfluck** [comment]
Update: thanks to @freddie on discord, I solved this one by changing to
'&:focus-visible': {
            outlineWidth: value,
            outlineStyle: 'solid',
            outlineColor: 'token(colors.blue.500)',
            outlineOffset: value,
},
I still think this is worth for the mantainers to take a look, as the "default" sintax doesn't seem to work here. I mean the _focusVisible and the color token without the token() wrapping the value.

**Reply by anubra266** [reply]
@FilipePfluck 
- Utilities should resolve to raw css objects. We convert them straight to css without processing them. SO `focusVisible` or other conditions can't work there
- Outline color works fine
![CleanShot 2023-08-07 at 13 43 55@2x](https://github.com/chakra-ui/panda/assets/30869823/7f17e60a-ab72-4c4a-9767-2b2f7edfb2f9)
